CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Open Source Studio at ITP
+
+## Who can contribute?
+**Everyone** is welcome to engage in the open source project that is the organisation you are currently viewing. We encourage discussion and contribution from outside the student body of the class. 
+
+The only limitation as an outside collaborator is the actual class meeting space, a seminar-style meeting in person. This is a private space for the enrolled students to discuss, question, and collaborate on assignments around the practice and philosophy of open source.
+
+## How to contribute
+Please consider our [code of conduct](https://github.com/Open-Source-Studio-at-ITP/Playground/blob/source/CODE_OF_CONDUCT.md) in all your interactions with our project. 
```

Comment:
We should probably link directly to a Code of Conduct in this repo or else create one for all repos. I think in this case there is little change from repo to repo creating `Open-Source-Studio-at-ITP/Code-of-Conduct` probably makes sense?

Reply:
See #38, let's link to there from here.